tajo-plan/src/main/java/org/apache/tajo/plan/rewrite/rules/FilterPushDownRule.java

```diff
@@ -358,7 +353,21 @@ public LogicalNode visitJoin(FilterPushDownContext context, LogicalPlan plan, Lo
       context.pushingDownFilters.removeAll(matched);
     }
 
-    context.pushingDownFilters.addAll(outerJoinFilterEvalsExcludePredication);
+    matched.clear();
+    matched.addAll(outerJoinFilterEvalsExcludePredication);
```

Comment:
I know that you didn't name the variables ```outerJoinPredicationEvals``` and ```outerJoinFilterEvalsExcludePredication```.

Their name is not intuitive. So, it would be great if we rename them to more better ones.